Nodejs/Product/Nodejs/Project/DependencyNode.cs

```diff
@@ -28,54 +28,25 @@
 using VsCommands2K = Microsoft.VisualStudio.VSConstants.VSStd2KCmdID;
 
 namespace Microsoft.NodejsTools.Project {
-    internal class DependencyNode : HierarchyNode {
+    internal interface IDependencyNodeBasePath {
+        string GetRelativeUrlFragment();
+    }
```

Comment:
what's the purpose of this interface? It's only referenced once and the value-add is unclear.